pyuvdata/tests/test_uvfits.py

```diff
@@ -129,9 +143,23 @@ def test_multisource_error():
 
         par_names = vis_hdu.data.parnames
         group_parameter_list = []
-        # need to account for PZERO values
+
+        lst_ind = 0
         for index, name in enumerate(par_names):
-            group_parameter_list.append(vis_hdu.data.par(name) - vis_hdr['PZERO' + str(index + 1)])
+            par_value = vis_hdu.data.par(name)
+            # lst_array needs to be split in 2 parts to get high enough accuracy
+            if name.lower() == 'lst':
+                if lst_ind == 0:
+                    # first lst entry, par_value has full lst value (astropy adds the 2 values)
```

Comment:
From the comment here, it sounds like astropy handles splitting the array into two entries. Have we checked that the CASA uvfits reader can handle this? Because the uvfits reader is one of the only ways to import pyuvdata objects into CASA, I'd like to check it works before merging this in.

Reply:
We should check. It is the standard way to handle the time array, although I have seen files that don't do this. Also, I'm not sure if CASA will even read this because it's not one of the standard arrays. Do you know who is working with CASA and could do this test for us?

Reply:
I have read uvfits into CASA before, but don't have experience with ensuring the equality of two different measurement sets. @mkolopanis or @nkern may be able to offer some insight.

Reply:
It's a little tough to see, and when I say a little I mean almost impossible but CASA keeps crashing on me. I read the file `day2_TDEM0003_10s_norx_1src_1spw.uvfits` and wrote out in this branch. Here is a plot the amplitudes of the first file and the file written by this branch. The blue dots are the before, the red diamonds are the file written by this branch, if you zoom they mach up to the eye's precision at least. 

It doesn't _seem_ to cause an issue with casa's reader, but I bet there is a more rigorous test we could make too.

![image](https://user-images.githubusercontent.com/8646829/57322813-56db8b00-70b9-11e9-96e6-5bec9cf66345.png)

Reply:
@plaplant does this meet your needs or do you want something more extensive?

Reply:
Thanks @mkolopanis. Yes, I think this is sufficient. I was mostly concerned about CASA choking entirely and not importing the data, so I'm satisfied.